tests/shouldwork/Cores/Xilinx/Floating/TH.hs

```diff
@@ -536,6 +522,30 @@ nanTest =
     , (nan, nan, F.xilinxNaN)
     ]
 
+fromUBasicSamples :: [(Unsigned 32, Float)]
+fromUBasicSamples = delayOutput0 (natToNum @F.FromU32DefDelay) $
+  map (\x -> (x, fromIntegral x))
+    [ 0
+    , 1
+    , maxBound
+    , 0b0010_1010_1010_1010_1010_1010_1010_1010
+    , 0b0101_0101_0101_0101_0101_0101_0101_0101
+    , 0b1010_1010_1010_1010_1010_1010_1010_1010
+    , 0b1101_0101_0101_0101_0101_0101_0101_0101
```

Comment:
These are just picked at random I suppose?

Reply:
No... they are two bitpatterns (the classic 0x55 and 0xaa) but with special treatment for what would have been the sign bit if it were signed. I'm trying to suss out bugs in Vivado, thinking adverserially, "well it is _supposed_ to be unsigned, but is it really?".

Reply:
Ah ok, gotcha. It didn't have a comment which is why I figured they were more or less randomly picked.

Reply:
I commented extensively.